src/Common.UnitTests/Handlers/PublishPackageHandlerTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Cmf.CustomerPortal.Sdk.Common;
+using Cmf.CustomerPortal.Sdk.Common.Handlers;
+using Cmf.CustomerPortal.Sdk.Common.Services;
+using Cmf.Foundation.BusinessObjects.QueryObject;
+using Cmf.Foundation.BusinessOrchestration.QueryManagement.OutputObjects;
+using Moq;
+using System.Reflection;
+
+namespace Common.UnitTests.Handlers;
+
+public class PublishPackageHandlerTests
+{
+    [Fact]
+    public async void Run_PackageDoesNotExistInPortal_PackageIsUploaded()
+    {
+        // arrange
+        var customerPortalClientMock = new Mock<ICustomerPortalClient>();
+        var sessionMock = new Mock<ISession>();
+        var queryProxyServiceMock = new Mock<IQueryProxyService>();
+        var publishPackageHandler = new PublishPackageHandler(customerPortalClientMock.Object, sessionMock.Object, queryProxyServiceMock.Object);
+        var fileSystemInfo = new FileInfo(Path.Combine(
+                                        Directory.GetParent(Assembly.GetExecutingAssembly().Location)!.FullName,
+                                        "assets",
+                                        "Cmf.Environment.11.2.0-alpha.1.zip"));
```

Comment:
move this filename to const. Its used in multiple places :)
Also we can think about do the same for assets folder